src/test/java/com/google/sps/servlets/getArticlesServletTest.java

```diff
@@ -0,0 +1,169 @@
+package com.google.sps.servlets;
```

Comment:
Minor style issue: For all classes we declare package first, then static imports, then imports. Also add a empty line between them. Example:

package a.b.c;

import static ....;
import static ....;

import ...;
import ...;

Try format file and see if it is auto fixed. Google internal tool can do this, but not so sure about GCP

Reply:
Sure! Will check that out. Thanks for pointing that out!